collateral-onboarding/hardhat/hardhat.config.js

```diff
@@ -0,0 +1,27 @@
+require('dotenv').config();
+require('@nomiclabs/hardhat-waffle');
+
+const ALCHEMY_URL = process.env.ALCHEMY_URL;
+
+if (!ALCHEMY_URL) {
+    throw new Error('ALCHEMY_URL env variable not set, please check your `hardhat/.env` file');
+}
+
+/**
+ * @type import('hardhat/config').HardhatUserConfig
+ */
+module.exports = {
+    solidity: '0.8.4',
+    networks: {
+        hardhat: {
+            chainId: 1337,
+            forking: {
+                url: ALCHEMY_URL,
+                // Few blocks before WSTETH-A is taken at 14052147,
+                // https://etherscan.io/address/0x49a33a28c4c7d9576ab28898f4c9ac7e52ea457a
+                blockNumber: 14052140,
```

Comment:
Why this specific block number? I imagine that we should actually fork without it to be on the latest state of the maker network

Otherwise, for now, you can take the block number where `MCD_CLIP_WSTETH_A` didn't exist yet (ie [`13426393 - 1`](https://etherscan.io/address/0x49a33a28c4c7d9576ab28898f4c9ac7e52ea457a#internaltx)) and try to execute the spell which introduced it, then check if `MCD_CLIP_WSTETH_A` exists in the `chainlog` contract (which should at this point)

Reply:
well, there's no reason for a specific block number: 

- in order to test that e.g. the MCD_CLIP_WSTETH_A , i would also have to checkout different commit there
- because currently there is no collateral being introduced by the latest version of the spell.

 also why i claimed that this runs without failure and not that it works.

Reply:
> there's no reason for a specific block number
> i would also have to checkout different commit there

Well, I still think that it makes sense to fork on a specific block number and maybe (as you suggest above) run a spell on a specific commit, in order to validate that contract that was not available is available after the execution

Reply:
So the takeaway here is the additional test setup that might need to be created